src/apps/assets/sass/components/_font-button.scss

```diff
@@ -23,6 +23,14 @@
     }
 }
 
+.btn-edit {
+    margin-left: 58%;
+    margin-top: 7px;
+    width: 100px;
+    font-size: $font-size-large;
+    margin-bottom: 7px;
+}
```

Comment:
let's make the button float on the right side of the editor, do this instead
- on parent div add a `position: relative`
```sass
.btn-edit {
    position: absolute;
    top: 25px;
    right: 25px;
}
```